packages/devextreme/js/__internal/grids/grid_core/column_fixing/m_column_fixing.ts

```diff
@@ -1148,6 +1149,18 @@ const resizing = (Base: ModuleType<ResizingController>) => class ResizingColumnF
   }
 };
 
+const keyboardNavigation = (Base: ModuleType<KeyboardNavigationController>) => class KeyboardNavigationExtender extends Base {
+  protected _toggleInertAttr(value = false): void {
+    const $fixedContent = this._rowsView?.element()?.children(`.${this.addWidgetPrefix(CONTENT_FIXED_CLASS)}`);
```

Comment:
Let's add to the ```rowsView``` extender (in the```m_column_fixing```) the method that returns fixed content element.

Something like:
```ts
const rowsView = (Base: ModuleType<RowsView>) => class RowsViewFixedColumnsExtender extends baseFixedColumns(Base) {
  // other methods & fields
  public getFixedContentElement(): dxElementWrapper {
    const fixedContentClass = this.addWidgetPrefix(CONTENT_FIXED_CLASS);
    return this.element()?.children(`.${fixedContentClass}`);
```
And use this method here:
```suggestion
    const $fixedContent = this._rowsView?.getFixedContentElement();
```